app/Commands/DisableCommand.php

```diff
@@ -67,29 +72,44 @@ public function disableByServiceName(string $service): void
 
                 return;
             case 1:
-                $serviceContainerId = $serviceMatches->flip()->first();
+                $this->disableByContainerId($serviceMatches->flip()->first());
```

Comment:
Is there a reason to refactor this to move the actual disabling into the methods, rather than keeping it like it was before where each case set `$serviceContainerId` and then the bottom handled it?

Reply:
Not that I can tell.  Might have ended up like that as I was coming up with the PR.  I have updated the code and will push shortly.